### DISCUSSION TITLE
Allow the ignores tab to work in specific chats like the filters do

### DISCUSSION THREAD

**Discussion by N8teAnderson** [Chatterino/chatterino2#3237] [Ideas]
Can we please add the ability to block or replace certain words in specific chats?

**Comment by floodybYte** [comment]
You can do that with filters

**Reply by N8teAnderson** [reply]
how? so far all i thought filters were was to search through the chat instead of change it clientside

**Comment by Felanbird** [comment]
> 
> 
> how? so far all i thought filters were was to search through the chat instead of change it clientside

You can block the entire message when it contains a certain phrase with filters, but not replace.

!(message.content contains "asd123") or !(message.content match r"regex_here")
